### DISCUSSION TITLE
Where can I find the default WebUI password?

### DISCUSSION THREAD

**Discussion by roverso** [VueTorrent/VueTorrent#1720] [Q&A] [answered]
Hello, I'm running VueTorrent on Windows. I managed to get all set up but I can't find the log file to find out what the password to access the interface is. I'm running with npm start not through container. General Node logs in AppData don't show the password.

**Comment by Larsluph** [comment, marked as answer]
Default username is always: `admin`
Since 4.6.1, default password `adminadmin` has been deprecated for security purposes, explanations can be found in the PR: qbittorrent/qBittorrent#19777

## On Desktop

By default, I believe the WebUI server ***isn't created by qBittorrent*** when using the desktop version of qBittorrent.
You should see a log entry saying `WebUI: Credentials are not set`.

Please set them in the qBittorrent settings > WebUI > Authentication.

## On NOX version

When using qBittorrent-NOX, ***a random password is generated on launch*** when it hasn't been set to prevent locking you out of the app.
The generated password is displayed **inside the stdout** of the associated process of qBittorrent.

**Reply by vdbhb59** [reply]
Sorry, but in my nox version the stdout just showed as adminadmin and it still does not work. How do I access my setup now? Pointed the same in my current issue as well (404, abrupt + login fail here: #1914)

**Reply by ShreshthTiwari** [reply]
> Default username is always: `admin` Since 4.6.1, default password `adminadmin` has been deprecated for security purposes, explanations can be found in the PR: [qbittorrent/qBittorrent#19777](https://github.com/qbittorrent/qBittorrent/pull/19777)
> ## On Desktop
> 
> By default, I believe the WebUI server _**isn't created by qBittorrent**_ when using the desktop version of qBittorrent. You should see a log entry saying `WebUI: Credentials are not set`.
> 
> Please set them in the qBittorrent settings > WebUI > Authentication.
> ## On NOX version
> 
> When using qBittorrent-NOX, _**a random password is generated on launch**_ when it hasn't been set to prevent locking you out of the app. The generated password is displayed **inside the stdout** of the associated process of qBittorrent.

Looks like there is some issue

<img width="1219" height="206" alt="image" src="https://github.com/user-attachments/assets/b8ef01e8-9804-47b9-842f-08d8ec0ba0d7" />
<img width="1290" height="1045" alt="image" src="https://github.com/user-attachments/assets/33f51316-22e4-4996-9be9-823503f9f1d3" />

**Reply by Larsluph** [reply]
@ShreshthTiwari Are you using the backend or only the UI ?
Try accessing qBittorrent logs, they log every login attempt with the reason why it was rejected.
Basically, 403 means banned and 401 means failure (usually bad credentials but it can be HTTP misconfiguration).

**Comment by roverso** [comment]
Thank you! Solved!
